### PR TITLE
Tests and code style

### PR DESCRIPTION
**Pull request by AlexanderLitus**

### CODE REVIEW

**Comment by AlexanderLitus** [comment]
PTAL.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <option name="m_regex" value="s_[a-z][A-Za-z\d]*" />
+      <option name="m_minLength" value="3" />
+      <option name="m_maxLength" value="32" />
+    </inspection_tool>
+    <inspection_tool class="LoggerInitializedWithForeignClass" enabled="false" level="WARNING" enabled_by_default="false">
+      <option name="loggerClassName" value="org.apache.log4j.Logger,org.slf4j.LoggerFactory,org.apache.commons.logging.LogFactory,java.util.logging.Logger" />
+      <option name="loggerFactoryMethodName" value="getLogger,getLogger,getLog,getLogger" />
+    </inspection_tool>
+  </profile>
+</component>
```
Comment (armiol) [inline .idea/inspectionProfiles/Project_Default.xml]: new line?

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="PROJECT_PROFILE" value="Project Default" />
+    <option name="USE_PROJECT_PROFILE" value="true" />
+    <version value="1.0" />
+  </settings>
+</component>
```
Comment (armiol) [inline .idea/inspectionProfiles/profiles_settings.xml]: new line?

File: protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/Extension.groovy [hunk 3]
```diff
@@ -28,127 +28,128 @@ import static java.util.Collections.singletonList
...
             return path
         }
     }
 
-    public static String getMainDescriptorSetPath(Project project) {
-        final String path = project.spineProtobuf.mainDescriptorSetPath
-        if (path == null) {
+    static GString getMainDescriptorSetPath(Project project) {
+        final GString path = project.spineProtobuf.mainDescriptorSetPath
+        if (!path) {
```
Comment (armiol) [inline protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/Extension.groovy]: What if it is empty?

File: protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/failures/FailureWriter.groovy [hunk 4]
```diff
@@ -127,20 +126,19 @@ class FailureWriter {
...
-        final Set<Map.Entry<String, String>> fieldsEntries = readFieldValues().entrySet()
+        final Set<Map.Entry<GString, GString>> fieldsEntries = readFieldValues().entrySet()
         for (int i = 0; i < fieldsEntries.size(); i++) {
-            Map.Entry<String, String> field = fieldsEntries.getAt(i)
-
+            final Map.Entry<GString, GString> field = fieldsEntries.getAt(i)
             writer.write("${field.value} ${getJavaFieldName(field.key, false)}")
-
-            if (i != fieldsEntries.size() - 1) {
+            final boolean isNotLast = i != fieldsEntries.size() - 1
```
Comment (armiol) [inline protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/failures/FailureWriter.groovy]: Consider enclosing the right part with `()` for better readability.

File: protobuf-plugin/plugin/src/test/groovy/org/spine3/gradle/ProtobufPluginShould.groovy [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+import static org.spine3.gradle.Given.*
+
+@SuppressWarnings("GroovyInstanceMethodNamingConvention")
+class ProtobufPluginShould {
+
+    private Project project
+    private TaskContainer tasks
+
+    static {
+        /** Returns `true` if a task depends on a passed one, `false` otherwise. */
```
Comment (armiol) [inline protobuf-plugin/plugin/src/test/groovy/org/spine3/gradle/ProtobufPluginShould.groovy]: Remove extra `*` or use line comment style (`//`).

File: reflections-plugin/plugin/src/main/groovy/org/spine3/gradle/ReflectionsPlugin.groovy [hunk 6]
```diff
@@ -42,41 +42,38 @@ class ReflectionsPlugin implements Plugin<Project> {
...
+    }
 
-            config.setUrls(urls)
+    private static void scanClassPath(Project project) {
+        final GString outputDir = "${project.projectDir}/build"
+        final File outputFile = new File(outputDir)
+        outputFile.mkdirs()
 
-            config.setScanners(new SubTypesScanner(), new TypeAnnotationsScanner());
+        final GString reflectionsOutputDir = "${project.projectDir}/src/generated/resources/META-INF/reflections"
```
Comment (armiol) [inline reflections-plugin/plugin/src/main/groovy/org/spine3/gradle/ReflectionsPlugin.groovy]: How do you know the path is `.../src/generated/...`? Isn't that configurable in the build script somewhere?

**Comment by armiol** [comment]
@AlexanderLitus please see my comments.

File: protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/Extension.groovy (hunk 3 above)
Comment (AlexanderLitus) [inline protobuf-plugin/plugin/src/main/groovy/org/spine3/gradle/Extension.groovy]: It is like in JS, if the string is null or empty, it is false when convert to boolean.

File: protobuf-plugin/plugin/src/test/groovy/org/spine3/gradle/ProtobufPluginShould.groovy (hunk 5 above)
Comment (AlexanderLitus) [inline protobuf-plugin/plugin/src/test/groovy/org/spine3/gradle/ProtobufPluginShould.groovy]: This is one-line Javadoc we use if the doc is short.

**Comment by AlexanderLitus** [comment]
@armiol PTAL.

**Comment by armiol** [comment]
LGTM.

**Pull request merged**